jupyter_server/services/sessions/sessionmanager.py

```diff
@@ -243,14 +244,21 @@ async def session_exists(self, path):
                 exists = True
         return exists
 
-    def new_session_id(self):
+    def new_session_id(self) -> str:
         "Create a uuid for a new session"
         return str(uuid.uuid4())
 
     async def create_session(
         self, path=None, name=None, type=None, kernel_name=None, kernel_id=None
     ):
-        """Creates a session and returns its model"""
+        """Creates a session and returns its model
+
```

Comment:
Should we add parameter descriptions for the others, or remove the description for `name`?

Reply:
I added the description of what I was sure of, but yes in general many of those function should get actual documentation.